apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java

```diff
@@ -180,44 +180,41 @@ public void process(
       Consumer<Throwable> errorHandler,
       Consumer<RifRecordLoadResult> resultHandler) {
 
-    BlockingThreadPoolExecutor loadExecutor = createLoadExecutor(options);
+    final RifFileType fileType = dataToLoad.getSourceEvent().getFile().getFileType();
+    final LoadAppOptions.PerformanceSettings performanceSettings =
+        options.selectPerformanceSettingsForFileType(fileType);
+    final BlockingThreadPoolExecutor loadExecutor = createLoadExecutor(performanceSettings);
 
-    MetricRegistry fileEventMetrics = dataToLoad.getSourceEvent().getEventMetrics();
-    Timer.Context timerDataSetFile =
+    final MetricRegistry fileEventMetrics = dataToLoad.getSourceEvent().getEventMetrics();
+    final Timer.Context timerDataSetFile =
         appState
             .getMetrics()
             .timer(MetricRegistry.name(getClass().getSimpleName(), "dataSet", "file", "processed"))
             .time();
     LOGGER.info("Processing '{}'...", dataToLoad);
 
-    dataToLoad
-        .getSourceEvent()
-        .getEventMetrics()
-        .register(
-            MetricRegistry.name(getClass().getSimpleName(), "loadExecutorService", "queueSize"),
-            new Gauge<Integer>() {
-              /**
-               * @see com.codahale.metrics.Gauge#getValue()
-               */
-              @Override
-              public Integer getValue() {
-                return loadExecutor.getQueue().size();
-              }
-            });
-    dataToLoad
-        .getSourceEvent()
-        .getEventMetrics()
-        .register(
-            MetricRegistry.name(getClass().getSimpleName(), "loadExecutorService", "activeBatches"),
-            new Gauge<Integer>() {
-              /**
-               * @see com.codahale.metrics.Gauge#getValue()
-               */
-              @Override
-              public Integer getValue() {
-                return loadExecutor.getActiveCount();
-              }
-            });
+    fileEventMetrics.register(
+        MetricRegistry.name(getClass().getSimpleName(), "loadExecutorService", "queueSize"),
+        new Gauge<Integer>() {
+          /**
+           * @see com.codahale.metrics.Gauge#getValue()
```

Comment:
Can these just be inheritDocs instead of sees?

Reply:
In fact.  Best to remove the redundant comment entirely if we want to change this existing code in this PR.

Reply:
I'll just convert the anonymous inner class to a lambda to remove the extra noise.